platform-sdk/build.gradle.kts

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-plugins { id("com.hedera.hashgraph.root") }
+plugins { id("com.hedera.hashgraph.platform-root") }
```

Comment:
A bit unlucky that we need this special handling now and need to have different "lifecycles" for the two projects. But I assume that can change again in the future when the publishing gets more unified over both projects.

Reply:
Yeah, however I think this will improve once the Hedera Node jars are published to MC and JFrog Artifactory. We are launching JFrog here in the next couple of weeks so this should get cleaner as we integrate with JFrog.